site/netlify.toml

```diff
@@ -22,3 +22,28 @@ status = 301
 from = "/*/guide/inline-queries.html"
 to = "/:splat/plugins/inline-query.html"
 status = 301
+
+[[redirects]]
+from = "/*"
+to = "/404.html"
+status = 404
+
+[[redirects]]
+from = "/es/*"
+to = "/es/404.html"
+status = 404
+
+[[redirects]]
+from = "/id/*"
+to = "/id/404.html"
+status = 404
+
+[[redirects]]
+from = "/uk/*"
+to = "/uk/404.html"
+status = 404
+
+[[redirects]]
+from = "/zh/*"
+to = "/zh/404.html"
+status = 404
```

Comment:
Are these configs correct? I got them from https://docs.netlify.com/routing/redirects/redirect-options/?#custom-404-page-handling. But it seems the 404 page is still not being redirected. What's wrong? :thinking: